Add FAQ to README and add hubble and tetragon folders (#60)
cilium/design-cfps: README.md

at any time if there is interest in the project. The next step for a dormant CFP is to amend it to be Implementable.

5. Declined: This proposal was considered by the community but ultimately rejected. The community may come back to the proposal in the future but a new CFP should be used. Rejected proposals can be useful as documentation on why the given proposal did not make sense.

## Other resources 

> Need some more clairification? Here are a list of resources on the Cilium community, and some FAQs on how to get more involved! Or please reach out on the Cilium Slack for other questions.

Reviewer: ```suggestion
Need more clarification? Here are a list of resources from the Cilium community, and some FAQs on how to get more involved! Can't find your answer here? Please reach out on the Cilium Slack [#development](https://cilium.slack.com/archives/C2B917YHE) for other questions.
```

### FAQs

<details>
<summary>Who can review a CFP?</summary>

>     Anyone can leave a helpful review on a CFP! Any questions, points of clarification that can help refine a CFP is greatly appreciated.

Reviewer: ```suggestion
    Anyone can leave a helpful review on a CFP! Questions, ideas, and points of clarification that can help refine a CFP are greatly appreciated.
```

</details>

<details>
<summary>Who can approve a CFP?</summary>

>     In order to approve a CFP, an Approver in the Cilium community must review a CFP submitted. It is expected that a [Code Team](https://github.com/cilium/community/blob/main/CONTRIBUTOR-ROLES.md#code-team) can approve a CFP.

Reviewer: ```suggestion
    Any [committer](https://github.com/cilium/cilium/blob/main/MAINTAINERS.md) can approve a CFP.
```

</details>

<details>

> <summary>How to gauge interest on my CFP?</summary>

Reviewer: ```suggestion
<summary>How can I gauge interest on my CFP?</summary>
```

>     Attend a Cilium weekly meeting or ask on the Cilium Slack for getting preliminary feedback on your CFP. If possible, ask in the relevant [SIG](https://docs.cilium.io/en/stable/community/community/#special-interest-groups).

Reviewer: ```suggestion
    To get preliminary feedback on your CFP, attend a Cilium weekly [community meeting](https://docs.cilium.io/en/stable/community/community/#community-meetings) or ask on the Cilium Slack . If possible, ask in the relevant [SIG](https://docs.cilium.io/en/stable/community/community/#special-interest-groups) slack channel or meeting.
```
